pkgs/development/libraries/glibc/common.nix

```diff
@@ -134,6 +134,13 @@ stdenv.mkDerivation ({
         url = "https://patchwork.sourceware.org/project/glibc/patch/20220314175316.3239120-2-sam@gentoo.org/raw/";
         sha256 = "sq0BoPqXHQ69Vq4zJobCspe4XRfnAiuac/wqzVQJESc=";
       })
+
+      /* Patch pending upstream inclusion to fix string.h syntax for older gcc.
+         Needed to unbreak gnat bootstrap against old gcc in nixpkgs. */
```

Comment:
Can you link https://patchwork.sourceware.org/project/glibc/patch/20220520150609.346566-1-slyfox@gentoo.org/ here? Was a bit hard to find the context initially.

Reply:
Sounds good! Added as :

```diff
@@ -139 +139,2 @@ stdenv.mkDerivation ({
-         Needed to unbreak gnat bootstrap against old gcc in nixpkgs. */
+         Needed to unbreak gnat bootstrap against old gcc in nixpkgs:
+           https://patchwork.sourceware.org/project/glibc/patch/20220520150609.346566-1-slyfox@gentoo.org/ */
```